src/pages/academy/blog/_all-articles.tsx

```diff
@@ -14,21 +16,47 @@ type AllArticlesProps = {
 export type ItemType = ArticleDataType[0]
 
 const AllArticles = ({ article_data }: AllArticlesProps) => {
+    const [is_mobile] = useBrowserResize()
+    const [is_tablet] = useBrowserResize(1333)
+    const [current_page, setCurrentPage] = useState(1)
+    const desktop_max_posts = 19
```

Comment:
Since we have 3 columns of videos in desktop, the number should be divisible to 3 to have well aligned rows. Therefor I updated it to 18. 19 was a mistake there, thanks for mentioning.